### PR TITLE
Updated a bunch of the colors, added an image or two, and a bunch of …

### LINKED ISSUE #36: Change all static text/images to be relevant to CharacterVault

### PR DESCRIPTION
**Pull request by DColePetty**
…text on the page.
closes #36

### CODE REVIEW

File: characterdapp/src/components/CreateZombie.js [hunk 1]
```diff
@@ -99,4 +100,4 @@ class CreateZombie extends Component {
   }
 }
 
-export default connect(mapStateToProps)(CreateZombie);
\ No newline at end of file
+export default connect(mapStateToProps)(CreateZombie);
```
Comment (instinctual07) [inline characterdapp/src/components/CreateZombie.js]: CreateSheet

File: characterdapp/src/components/TopBar.js [hunk 2]
```diff
@@ -22,35 +22,35 @@ class TopBar extends Component {
...
         </Menu>
         <div className="center">
-          <h2>The most awesome zombies in the whole wide universe!</h2>
+          <h2>The place where your imagination is forever!</h2>
         </div>
-        Your account address: {this.props.userAddress}
+        Your current account address: {this.props.userAddress}
         <br />
-        You own {this.props.userZombieCount} zombie(s) out of a total of approximately {this.props.totalZombieCount}.
+        You have {this.props.userZombieCount} character sheets out of a {this.props.totalZombieCount} in the vault.
```
Comment (instinctual07) [inline characterdapp/src/components/TopBar.js]: userSheetCount

File: characterdapp/src/utils/initBlockchain.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import CryptoZombiesContract from "../contract_ABI/CryptoZombies.sol/CryptoZombies.json";
+//import CryptoZombiesContract from "../contract_ABI/CharacterVault.sol/CharacterVault.json";
```
Comment (instinctual07) [inline characterdapp/src/utils/initBlockchain.js]: CharacterVaultContract

**Review by instinctual07 — CHANGES_REQUESTED** [review]
A few "zombie" misses here. I've just been search and replacing instances of "zombie,Zombie,ZOMBIE" with sheet. And "CZ" to "CV" when applicable

File: characterdapp/src/utils/initBlockchain.js (hunk 3 above)
Comment (DColePetty) [reply characterdapp/src/utils/initBlockchain.js]: import CharacterVaultContract from "../contract_ABI/CharacterVault.sol/CharacterVault.json";
^ like so?

**Comment by DColePetty** [comment]
Ok all those changes were made, please re-review

**Review by instinctual07 — APPROVED** [review]

**Pull request closed without merging**